contracts/javascore/xcall/src/test/java/foundation/icon/btp/xcall/CallServiceTest.java

```diff
@@ -0,0 +1,237 @@
+package foundation.icon.btp.xcall;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.verify;
+
+import foundation.icon.btp.xcall.data.CSMessageResponse;
+import ibc.icon.interfaces.ICallServiceReceiverScoreInterface;
+import java.math.BigInteger;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.function.Executable;
+import org.mockito.MockedConstruction;
+import org.mockito.Mockito;
+import score.UserRevertedException;
+
+
+public class CallServiceTest extends CallServiceTestBase {
+
+    @BeforeEach
+    public void setup() throws Exception {
+        super.setup();
+        client.invoke(owner, "setTimeoutHeight", TIMEOUT_HEIGHT);
+    }
+
+    @AfterEach
+    protected void teardown() {
+        super.teardown();
+    }
+
+
+    @Test
+    @Order(0)
```

Comment:
Order here can be remove since they are tested on separate deployments

Reply:
removed on new commit